ros/src/computing/perception/localization/packages/ekf_localizer/README.md

```diff
@@ -0,0 +1,85 @@
+# Overview
+This package generates robust self_pose/twist integrating pose and twist messages with vehicle model.
+
+Some other features include
+ - time delay compensation : for sensor delay, predict current position from vehicle dynamics. The delay is usually calculated by comparing the header time stamp with the current time, but additional delay time can be set with the parameter.
+ - mahalanobis gate : for sensor outlier detection, Mahalanobis distance is used.
+ - yaw angle compensation : yaw angle bias is estiamted, which is caused by sensor mounting error.
+
+
+# Input and Output
+- input
+    - geometry_msgs::PoseStamped ego_pose (default: `/ndt_pose`)
+    - geometry_msgs::TwistStamped ego_twist (default: `/can_twist`))
+    
+- output
+    - tf base_link : broadcasted with *tf_rate* [Hz]
+    - estimated pose (geometry_msgs::PoseStamped) : filtered pose
+    - estimated twist (geometry_msgs::TwistStamped) : filtered twist
+
+
+# Functions
+
+
+## in timer callback with *predict_frequency*
+Calculate *predict* and *update* at constant intervals in timer callback. The purpose of perform *update* at timer callback is to avoid large position (or velocity) changes due to low period measurement, such as NDT matching. If the callback period is faster than the sensor period, the same sensor value is used for update. Note that since the algorithm can deal with sensor time delay, there is no lack of consistency in using the same sensor data. 
+
+
+## measurement data callback
+
+The subscribed data is saved and used for *update* of kalman filter in timer callback. The Mahalanobis distance is calculated for the received data, and it is removed as an outlier if the value is over limit.
+
+
+# Parameter description
+
+they are set in `launch/ekf_localizer.launch` 
+
+
+## for Node
+
+|Name|Type|Description|
```

Comment:
can you please add a default value column?

Reply:
update readme.